src/routes/api/users.js

```diff
@@ -16,7 +19,7 @@ router.get('/user', (req, res, next) => {
 
 router.put('/user', (req, res, next) => {
   User.findById(req.payload.id)
```

Comment:
There shouldn't be logic exposed to the routes, please look into separating concerns.

Reply:
This part of the code wasn't implemented by me. I believe the changes were because of eslint fix that applied.

Reply:
Who wrote the code. Please find out.